File: maxMin.cpp
#include <iostream>
using namespace std;

void maxMin(int arr[], int n)
{
    int max = 0;
    int min = 987654321;
    for (int i = 0; i < n; i++)
    {
        while (arr[i] > max)
        {
            max = arr[i];
        }
        while (arr[i] < min)
        {
            min = arr[i];
        }
    }
    cout << "Max ele of arr : " << endl;
    cout << max << endl;
    cout << "Min ele of arr : " << endl;
    cout << min << endl;
}

int main()
{

    cout << "Enter Length Of Array : ";
    int n;
    cin >> n;
    int arr[n];
    cout << "Enter Array Elements :";
    for (int i = 0; i < n; i++)
    {
        cin >> arr[i];
    }
    maxMin(arr, n);
    return 0;
}

File: MaxMin.java
package com.company;
import java.util.Scanner;

public class MaxMin extends Object{

    public static void maMin(int arr[],int n){
        int max = -987654321;
        int min = 987654321;

        for(int i=0;i<n;i++){
            if(arr[i] > max){
                max = arr[i];
            }
        }
        System.out.println("Max : "+ max);

        for(int i=0;i<n;i++){
            if(arr[i] < min){
                min = arr[i];
            }
        }
        System.out.println("Min : "+ min);
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int arr[] = new int[n];
        for(int i=0;i<n;i++){
            arr[i] = sc.nextInt();
        }
        maMin(arr, n);
    }
}

